### PR TITLE
More installer related fixes

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Few more fixes related to blivet-gui in Anaconda.

### CODE REVIEW

File: blivetgui/list_devices.py [hunk 1]
```diff
@@ -71,6 +71,13 @@ def load_disks(self):
 
         disks = self.blivet_gui.client.remote_call("get_disks")
 
+        # hide protected and hidden disks in installer mode
+        # (this will hide USB drive with installation image, disks under FW RAID etc.)
+        if self.blivet_gui.installer_mode:
+            for disk in disks:
+                if disk.protected or disk.format.hidden:
+                    disks.remove(disk)
```
Comment (vpodzime) [inline blivetgui/list_devices.py]: Shouldn't this be done no matter what ``installer_mode`` says?

File: blivetgui/osinstall.py [hunk 2]
```diff
@@ -132,6 +132,12 @@ def __init__(self, client, spoke, spoke_container):
         self.physical_view = PhysicalView(self)
         self.builder.get_object("scrolledwindow_physical").add(self.physical_view.vbox)
 
+    def ui_refresh(self, _spoke):
+        # some configuration for ui elements needs to be done when the ui
+        # is actually visible -- in Anaconda we need to do this using the
+        # spoke "entered" signal
+        self.list_devices.disks_view.set_cursor(0)
```
Comment (vpodzime) [inline blivetgui/osinstall.py]: I don't really get what's being done here. A method is added, not called from anywhere and it does what seems to be basically a no-op.

Comment (vpodzime) [inline blivetgui/osinstall.py]: So this is hooked/called in/from Anaconda? I think this comment should be a proper docstring explaining what the method does and when it should be called.

Comment (vojtechtrefny) [inline blivetgui/osinstall.py]: Added better comment/docstring.

File: blivetgui/list_devices.py (hunk 1 above)
Comment (vojtechtrefny) [inline blivetgui/list_devices.py]: No, I do want to display all devices in blivet-gui, even if format is hidden or protected -- at least to let the user know there is a device with protected format.
I am doing this only in installer mode so the behavior is the same in blivet-gui and custom spoke.

Comment (vpodzime) [inline blivetgui/list_devices.py]: Fair enough

**Pull request merged**